resources/views/ws/ghap/places/show.blade.php

```diff
@@ -22,17 +22,22 @@
 
                 <a href="{{route('index')}}#advancedsearch" class="btn btn-secondary tlcmgreen" role="button" id="advancedsearch" title="Search within an area, exact or fuzzy matching and filter by attributes.">Advanced Search</a>
 
+                <!-- Adjust for route places/{id}/{format?} -->
+                @php
+                    $url = url()->full();
+                    $separator = (parse_url($url, PHP_URL_QUERY) == NULL) ? '?' : '&';
+                @endphp
 
                 <!-- Export/Download -->
                 <div class="dropdown">
                     <button class="btn btn-secondary dropdown-toggle tlcmgreen" type="button" id="downloadDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                         Download
                     </button>
                     <div class="dropdown-menu" aria-labelledby="downloadDropdown">
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=kml&download=on">KML</a>
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=csv&download=on">CSV</a>
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=json&download=on">GeoJSON</a>
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=rocrate&download=on">RO-Crate</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=kml&download=on">KML</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=csv&download=on">CSV</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=json&download=on">GeoJSON</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=rocrate&download=on">RO-Crate</a>

```

Comment:
Ok, now I see these links should work with multiple places result page as well. Please revert the change back as the redirection should handle that.